src/QtComponents/QtMgx3DMainWindow.cpp

```diff
@@ -5255,8 +5254,49 @@ log (t4);
 log (t5);
 */
 
-			COMPLETE_QT_TRY_CATCH_BLOCK(true, this, getAppTitle())
-		}    // QtMgx3DMainWindow::preferencesCallback
+	const TkUtil::Color	currentTopoColorWithoutProj (Internal::InternalPreferences::instance ( )._topoColorWithoutProj.getRed ( ), Internal::InternalPreferences::instance ( )._topoColorWithoutProj.getGreen ( ), Internal::InternalPreferences::instance ( )._topoColorWithoutProj.getBlue ( ));
+	const TkUtil::Color	currentTopoColorWith0DProj (Internal::InternalPreferences::instance ( )._topoColorWith0DProj.getRed ( ), Internal::InternalPreferences::instance ( )._topoColorWith0DProj.getGreen ( ), Internal::InternalPreferences::instance ( )._topoColorWith0DProj.getBlue ( ));;
+	const TkUtil::Color	currentTopoColorWith1DProj (Internal::InternalPreferences::instance ( )._topoColorWith1DProj.getRed ( ), Internal::InternalPreferences::instance ( )._topoColorWith1DProj.getGreen ( ), Internal::InternalPreferences::instance ( )._topoColorWith1DProj.getBlue ( ));;
+	const TkUtil::Color	currentTopoColorWith2DProj (Internal::InternalPreferences::instance ( )._topoColorWith2DProj.getRed ( ), Internal::InternalPreferences::instance ( )._topoColorWith1DProj.getGreen ( ), Internal::InternalPreferences::instance ( )._topoColorWith2DProj.getBlue ( ));;
```

Comment:
Same as above, 
there is a typo here for oldTopoColorWith2DProj where topoColorWith1DProj.getGreen is taken, it should be topoColorWith2DProj.getGreen